folsom/src/main/java/com/spotify/folsom/MemcacheClientBuilder.java

```diff
@@ -276,6 +278,29 @@ public MemcacheClientBuilder(final Transcoder<V> valueTranscoder) {
     return this;
   }
 
+  /**
+   * Specify the maximum number of operations that will be batched together in one network request.
```

Comment:
This is slightly confusing. Each operation is still a separate memcache request. It's just that the requests are written to the socket in a fewer writes (sometimes a single). Would it make sense to clarify this?